frontend/src/scenes/onboarding/sdks/sdk-install-instructions/android.tsx

```diff
@@ -33,6 +39,15 @@ function AndroidSetupSnippet(): JSX.Element {
             apiKey = POSTHOG_API_KEY,
             host = POSTHOG_HOST
         )
+        ${
+            includeReplay
+                ? `
+        config.sessionReplay = true
```

Comment:
Should we just point to this link though https://posthog.com/docs/session-replay/mobile#installation?
There are a few nuances such as limitations, the `PostHogOkHttpInterceptor` bits, etc, so they'd have more context on how it works and less code duplication here.